index.bs

```diff
@@ -324,6 +324,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Resident Key</dfn>
+: <dfn>RK</dfn>
+:: A [=resident key=] is a [=credential private key=] that can be found and used by the authenticator based on only the 
+    associated RP ID. 
+
```

Comment:
I consider the possibility of changing the name to "requireClientResidentKey:. However, most developers' perception of the word "client" is user agent, aka the browser. Using the name "client" may lead to developers thinking the private keys are kept on the browser. Therefore I propose we keep the name "requireResidentKey"